content/applications/hr/fleet/accidents.rst

```diff
@@ -0,0 +1,257 @@
+   :align: center
+   :alt: The Kanban view of stages, with a card being dragged and dropped to the Running stage.
+
+Accident reporting
+==================
+
+One of the main reasons to track accidents using the method outlined in this document is the ability
+to view the total accident cost, determine the safest drivers, and determine the actual total cost
+for specific vehicles.
+
+The main :ref:`Services dashboard <fleet/services_dashboard>` displays all the various accident
+information, while the :ref:`Reporting dashboard <fleet/reporting_dashboard>` displays the total
+cost for specific vehicles.
+
+.. _fleet/services_dashboard:
+
+Services dashboard
+------------------
+
+Navigate to :menuselection:`Fleet app --> Fleet --> Services` to view the default
+:guilabel:`Services` dashboard. All service records are displayed in a :icon:`oi-view-list`
+:guilabel:`(List)` view, grouped alphabetically, by :guilabel:`Service Type`.
+
+The two service types created for accident tracking appear in the list: :guilabel:`Accident -
+Driver Fault` and :guilabel:`Accident - No Fault`.
+
+Each grouping displays the number of records within each grouping, and lists the individual records
+beneath each grouping title.
+
+.. example::
+   In this example, there are six accidents where the driver was at fault, and four accidents that
+   were not the driver's fault. This default dashboard also displays the estimated total
+   :guilabel:`Cost` for all the accidents in each group.
+
+   There is an estimated `$19,164.81` dollars for driver-caused accident repairs, and an estimated
+   `$2,548.21` dollars for no-fault accidents.
+
+   .. image:: accidents/group-accidents.png
+      :align: center
+      :alt: Accident services, with the total costs highlighted.
+
+.. note::
+   The total :guilabel:`Cost` calculates **all** costs on the repair form, including estimated costs
+   as well as final repair costs. This number may not be accurate if there are any repairs in the
+   :guilabel:`Running` stage, and the final bill has not yet been calculated.
+
+.. _fleet/reporting_dashboard:
+
+Reporting dashboard
+-------------------
+
+Navigate to :menuselection:`Fleet app --> Reporting --> Costs` to view the default
+:menuselection:`Cost Analysis` report. The default report displays a  :icon:`fa-bar-chart`
+:guilabel:`(Bar Chart)` of all :guilabel:`Contract` and :guilabel:`Service` costs for the current
+year, organized by month (:guilabel:`Date`).
+
+To view the total cost by vehicle, click the :icon:`fa-caret-down` :guilabel:`(down caret)` icon at
+the far-right of the :guilabel:`Search...` bar, revealing a drop-down menu. Click
+:guilabel:`Vehicle` in the :icon:`oi-group` :guilabel:`Group By` column, and the data is organized
+by vehicle.
+
+This displays the true cost for each vehicle, including both the contract cost (such as the monthly
+vehicle lease cost) and all service costs, including all accidents. This allows for a more complete
+view of the vehicle cost.
```

Comment:
This is really useful information! How do I see it? 

I poked around and saw a breakdown of each cost type in the Pivot view. Were you referring to this? If so, I'd be great to specify that you got this info in the pivot view! :blush: 

![image](https://github.com/user-attachments/assets/1071282a-6683-413c-8a7d-e282c5de5819)